lib/osf-components/addon/components/search-result-card/styles.scss

```
@@ -1,25 +1,45 @@
 .result-card-container {
-    margin: 10px;
-
-    dt:not(:first-child) {
-        margin-top: 10px;
-    }
+    background-color: $color-bg-white;
+    margin: 35px 35px 0;
```

The dl, dt, and dd in the expanded section of the card could use margins as well. something like this should match the `h4` and `div` styled below
```suggestion
    margin: 35px 35px 0;
    dt,
    dd {
        margin-left: 15px;
    }
```
dd may need a separate `margin-bottom: 15px;`

---

The secondary metadata container is not currently rendering as dl, dt, or dd element tags. I have searched the main template code as well for cards and the primary route; would you like this change in place now or when we merge those changes for the result cards?

---

This is now added because Preprint styling is using defintition lists.